core/common/src/main/scala/net/liftweb/common/HList.scala

```diff
@@ -116,7 +96,7 @@ object HLists {
    * Carries the information about the type of this element, plus the `HList`
    * type of the rest of the list.
    *
-   * You can use `:+:` to make this `HList` longer:
+   * You can use `[[HListMethods.:+: :+:]]` to make this `HList` longer:
    *
    * {{{
```

Comment:
would you want to update the scaladoc style while you are here or are you planning on doing another massive pass at docs on a separate PR?